package.json

```diff
@@ -26,7 +26,8 @@
     "@actions/exec": "^1.1.0",
     "@actions/github": "^5.0.0",
     "@actions/io": "^1.1.1",
-    "@actions/tool-cache": "^1.7.1"
+    "@actions/tool-cache": "^1.7.1",
+    "@expo/eas-json": "0.48.0"
```

Comment:
I think it would make more sense if we could reuse the `$ eas config` command to extract the information we need. The reason for this is that the Github Action is prebuilt and is pretty static in the dependencies (including future updates). If we can just reuse whatever `eas-cli` version is installed for the project, we can push out updates without rebuilding the Expo Github Action.

Right now, the `eas config` command isn't usable for us, though. It doesn't allow you to output just JSON, and it doesn't output enough information for us to perform the actions. Would be awesome if we can update the `eas config` command to match the `expo config` command better, e.g. add JSON output and output info we can reuse.

Reply:
@EvanBacon suggested listing the available profiles that can be used to run the build command. So we need to read the eas.json file to find that information. And because the `eas config` command requires the parameter to be passed as a profile.

Reply:
Actually, its not implemented yet. So, I will do the way you suggest. 🖖